scripts/package_building_bisect/utils.sh

```diff
@@ -36,3 +40,22 @@ function get_job_number (){
     result="$(($cores * $multi))"
     echo ${result%.*}
 }
+
+get_destination_path2 () {
```

Comment:
this should never be done, just adding numbers to a method definition

you can just redefine it, think about it like class method override